x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -227,7 +226,7 @@ const MovingAveragePopup = () => {
       <p>
         <FormattedMessage
           id="xpack.lens.indexPattern.movingAverage.windowInitialPartial"
-          defaultMessage="For the initial part of the series the window is partial, until it reaches the requested number of items. For instance with a window size of 5:"
+          defaultMessage="The window is partial until it reaches the requested number of items.  For example, with a window size of 5:"
```

Comment:
The `initial` part was added to mark the specific case. Not a deal breaker for me, but check with @wylieconlon if he's ok with the change (he proposed the first version)